### ISSUE TITLE
Dots numbering is standardised?

### ISSUE THREAD

**Issue by sr12zar** [PyThaiNLP/thaibraille#4]
The number patterns used in Thaibraille (here) seems to be unique. Internationally standardised numbering as used in EU, UK, US for 6 dots (2 columns x 3 rows) is (1:top left, 2 middle left, 3 bottom left); (4 top right, 5 mid rt, 6 bot rt) and for 8 dots the 6 dots are shifted up and 2 more dots added as bottom row with numbers  (7 bot left; 8 bot rt).
  
Though the numbering in python code works now, there is a question on communication among international community.

**Comment by wannaphong** [comment]
I agree. Thaibraille doen't use standardised. I want to fellow Thai standardised but I don't see any standardised in my language (low resource for research), so I was create this project for create Thai Dots numbering that output is Braille Patterns block of the Unicode standard for printing.

**Comment by sr12zar** [comment]
Please have a look at https://github.com/sr12zar/KoKho/tree/main/script :
Bn2ThB.js and Th2Bn.js are javascript maps to international numbering
convention (that is 1237 4568 column-wise, dot 5 is still the second dot of
the second column). They should not be difficult to convert to Python.
And ../ThaiBraille.html from there is the html I used to test the maps. ;-)

As to the other issue (incorrect key: value pair), I corrected that in my
js maps.

I am about to add Braille and sign language scripts to one of the
educational games (to help learning many scripts used in Thailand). Perhaps
you can use them for something better.

Cheer
SR
